automation/ansible/variable_override.yml

```diff
@@ -41,3 +41,12 @@ r2_broker07_live_ip:      52.116.135.19
 r2_broker08_bak_ip:       52.118.96.216
 r2_vpc2_nfs_server_ip:    52.116.202.69
 r2_router03_spoke_ip:     52.116.129.90
+
+
+## Jmeter client setup variables.
+jmeter_binary_download_uri: https://dlcdn.apache.org/jmeter/binaries
+jmeter_binary_version: apache-jmeter-5.4.3
+jmeter_binary_archive: "{{ jmeter_binary_version }}.zip"
+jmeter_remote_bin_dir: jmeter
+jmeter_hub_router_ip: 163.75.95.11
```

Comment:
You can simply use following to setup the private IP and it won't need to change at all:
`jmeter_hub_router_ip: "{{ r1_vpc1_private_ip_prefix }}.128.100"`

Also, the jmeter properties should go in the `hosts.yml` as they should have some default values.

Reply:
Made the changes and I will be checking in soon.